src/mozmlops/templates/template_flow.py

```diff
@@ -60,11 +60,10 @@ def start(self):
             "WANDB_PROJECT": os.getenv("WANDB_PROJECT"),
         }
     )
-
     # You can uncomment and adjust this decorator to scale your flow remotely with a custom image.
     # Note: the image parameter must be a fully qualified registry path otherwise Metaflow will default to
     # the AWS public registry.
-    # The image referenced HERE is the mozmlops demo image, 
+    # The image referenced HERE is the mozmlops demo image,
     # which has both the dependencies you need run this template flow:
     # scikit-learn (for the specific model called in this demo) and mozmlops (for all your ops tools).
     @kubernetes(image="registry.hub.docker.com/chelseatroy/mozmlops:latest", cpu=1)
```

Comment:
Would you kindly check with SRE's (or with docker-etl owners!) how to publish to the company hubs? I'm not sure we can use personal accounts

Reply:
For Ray serve, docker images are being stored at [Artifact Registry](https://cloud.google.com/artifact-registry) of a team's/tenant's GCP project.

Reply:
Sweet 🙂 what I'd like to do is go ahead and merge _this_ change set and then scope moving the docker images to company platforms to a separate ticket and change set!